src/Resources/app/administration/src/module/blog-module/page/sas-blog-detail/index.js

```diff
@@ -41,6 +41,7 @@ Component.register('sas-blog-detail', {
             fileAccept: 'image/*',
             moduleData: this.$route.meta.$module,
             isProVersion: false,
+            slugBlog:null
```

Comment:
Please use space and comma reasonable

```suggestion
            slugBlog: null,
```